### DISCUSSION TITLE
Plotly Dash Interactive Mapping - Dash Leaflet and TiTiler

### DISCUSSION THREAD

**Discussion by mdavid800** [developmentseed/titiler#506] [Show and tell]
Hi Guys, 

I hope you don't mind but I have put together a blog post on a new website we are building for the use of Python in Wind Resource Assessment and Metocean. 

One issue we are having now in the offshore wind farm community is scale and I have found TiTiler really helps solve this problem.

I have made the article fairly lengthy to explain some of the background - as a lot of this is new to me / the community. It also includes reference to Rio-Cogeo, COGs, Explaination of using your Docker image etc. 

This package I think will be really useful to our community, so thanks for the great work. 

The link to the article can be found here : 

https://www.pywram.com/t/plotly-dash-interactive-mapping-dash-leaflet-titiler/287?u=mdavid800

Let me know if you have any issues with the article, I have tried my best to point back to the sources so you guys get the recognition of the hard work you have put in. 

David

**Comment by vincentsarago** [comment]
🙏 That's really cool @mdavid800
